zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java

```diff
@@ -381,8 +381,11 @@ String undoUnsuccessful(@PluralCount int unsuccessfulCount,
     @DefaultMessage("Page size")
     String pageSize();
 
-    @DefaultMessage("Read only")
-    String readOnly();
+    @DefaultMessage("This project-version is readonly. It cannot be edited.")
+    String readOnlyTooltip();
+
+    @DefaultMessage("This project-version is obsolete. It cannot be edited.")
```

Comment:
Except that Luke's been talking about renaming "read-only" as archived
(which makes a lot of sense if you don't worry about confusing users who
know our previous meaning for archived).

On 20 March 2015 at 09:43, David Mason <notifications@github.com> wrote:

> In
> zanata-war/src/main/java/org/zanata/webtrans/client/resources/WebTransMessages.java
> <https://github.com/zanata/zanata-server/pull/736#discussion_r26808785>:
>
> > @@ -381,8 +381,11 @@ String undoUnsuccessful(@PluralCount int unsuccessfulCount,
> >      @DefaultMessage("Page size")
> >      String pageSize();
> >
> > -    @DefaultMessage("Read only")
> > -    String readOnly();
> > +    @DefaultMessage("This project-version is readonly. It cannot be edited.")
> > +    String readOnlyTooltip();
> > +
> > +    @DefaultMessage("This project-version is obsolete. It cannot be edited.")
>
> Yes. It is obsolete in all the code, so I had that in mind. I'll change
> the string to "archived"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zanata/zanata-server/pull/736/files#r26808785>.
>